ee/tables/dataflattentable/tables.go

```diff
@@ -151,7 +159,7 @@ func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) (
 
 		for _, filePath := range filePaths {
 			for _, dataQuery := range tablehelpers.GetConstraints(queryContext, "query", tablehelpers.WithDefaults("*")) {
-				subresults, err := t.generatePath(ctx, filePath, dataQuery)
+				subresults, err := t.generatePath(ctx, filePath, dataQuery, append(flattenOpts, dataflatten.WithQuery(strings.Split(dataQuery, "/")))...)
```

Comment:
Why do we split the query on forward slash here and on 179?

Reply:
I'm not quite sure, I just copied it over form the [original code](https://github.com/kolide/launcher/blob/main/ee/tables/dataflattentable/tables.go#L175), I'll look into it a little more.

Reply:
Hmm searching the code for `dataflatten.WithQuery(strings.Split(dataQuery, "/")` reveals that we do this in many (all?) dataflatten tables, we should probably just bake that directly into dataflatten.